include/deal.II/base/symmetric_tensor.h

```diff
@@ -1217,14 +1217,14 @@ namespace internal
       case 2:
         return (data[0] * sdata[0] +
                 data[1] * sdata[1] +
-                Number(2.) * data[2] * sdata[2]);
```

Comment:
I don't think this works. To the best of my recollection, `std::complex<T>` only has a multiplication operator with the scalar `T`, but not with `double` or `int`. We had to change everything to `Number` in many many places over the years because of this.